osutil/fips/fips_test.go

```diff
@@ -0,0 +1,129 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package fips_test
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/osutil/fips"
+)
+
+func TestFIPSIsEnabled(t *testing.T) {
```

Comment:
Typically this would be a suite with the subtests as different `Test*` functions, so I guess I have the same question as with Zygmunt's parser test improvements. Are we using `testing` now when we feel like it's beneficial? We might end up having significantly different tests.
Pinging @pedronis so he can share his thoughts

Reply:
no, we are not going to use testing except for Benchmarks